cumulusci/robotframework/Salesforce.py

```diff
@@ -967,16 +968,15 @@ def salesforce_query(self, obj_name, **kwargs):
         else:
             query += "Id"
         query += " FROM {}".format(obj_name)
-        where = []
+        where_clauses = []
         if "where" in kwargs:
-            where = [kwargs["where"]]
-        else:
-            for key, value in kwargs.items():
-                if key == "select":
-                    continue
-                where.append("{} = '{}'".format(key, value))
-        if where:
-            query += " WHERE " + " AND ".join(where)
+            where_clauses = [kwargs["where"]]
```

Comment:
I wasn't expecting you to support both, I was thinking you would allow one way or the other. <shrug> I guess this is fine, though I'm not sure anyone will use it.

Reply:
It was approximately just as easy to support both and it enables one to add an extra != or < or > WHERE clause to code that already exists without rewriting it.